### ISSUE TITLE
Shippable.com CI support

### ISSUE THREAD

**Issue by trollixx** [badges/shields#355]
It would be great to have [shippable.com](https://www.shippable.com/) support. They have their own badges (below), but a unified version from shields.io would be nice too.

[![Build Status](https://api.shippable.com/projects/54ac2ce4d46935d5fbc19b84/badge?branchName=master)](https://app.shippable.com/projects/54ac2ce4d46935d5fbc19b84/builds/latest)

**Comment by espadrine** [comment]
Thanks for the suggestion, and the example!

Funnily enough, their badge looks eerily similar to the flat one that was designed in Shields.

They don't seem to have an API. This is the best source of information I have found on the subject: http://docs.shippable.com/en/latest/config.html#test-and-code-coverage-visualization

**Comment by trollixx** [comment]
They have API @ [api.shippable.com](https://api.shippable.com), but that is not much of use because of CORS. :(

**Comment by espadrine** [comment]
CORS is only an issue in the browser; we have a server. Do you have a link to the documentation for that API?

**Comment by trollixx** [comment]
It's not public according to Shippable/support#768.

**Comment by trollixx** [comment]
Here's a curl request to get all info about the latest build:

``` bash
curl 'https://api.shippable.com/projects/54ac2ce4d46935d5fbc19b84/builds/latest'
```

**Comment by espadrine** [comment]
Thanks a lot!

I think the honorable thing to do is to hold on while they make it public — which can probably be accelerated by voicing demand for it.

**Comment by trollixx** [comment]
I've just asked about ETA, let's see...

**Comment by larsbrinkhoff** [comment]
I think this was done in #462.

**Comment by espadrine** [comment]
Does it still work? The badges on the main website say `[ build | none ]`.

**Comment by larsbrinkhoff** [comment]
The SVG URL has changed.  It's now:
https://api.shippable.com/projects/5444c5ecb904a4b21567b0ff/badge?branch=master

**Comment by larsbrinkhoff** [comment]
Fixed in #696.

**Comment by espadrine** [comment]
I think the project ID changed again. It is a bit inconvenient, but hopefully the example is enough. Otherwise, I believe this issue is fixed. Feel free to re-open it if you disagree.
